front-end/src/components/NewGroup.vue

```diff
@@ -0,0 +1,72 @@
+<template>
+    <div class="popupcontainer">
+        <div class="fond"></div>
+        <form class="newGroupForm">
+            <button class="iconBig" type="button" @click="$emit('close')"><img src="https://cdn-icons-png.flaticon.com/512/2976/2976286.png" alt="Close"></button>
+            <div class="formInput">
+                <label for="name">Nom du groupe</label><br/>
+                <input type="text" name="name"><br/>
+            </div>
+            <div class="formInput">
+                <label for="members">Membres (entre 3 et 5)</label><br/>
+                <select type="text" name="members"><option></option></select>
+            </div>
+            <div class="formInput">
+                <label for="sprint">Sprint</label><br/>
+                <select type="text" name="sprint"><option></option></select>
+            </div>
+            <button>Créer un nouveau groupe</button>
+        </form>
+    </div>
+</template>
+
+<style>
+.popupcontainer {
```

Comment:
We do use tailwind instead of manual css.
But for the sake of not losing time it's fine we won't change it.